apps/frontend/src/pages/project/project.tsx

```diff
@@ -83,6 +85,20 @@ const Project = (): JSX.Element => {
 		if (projectId) {
 			void dispatch(projectActions.getById({ id: projectId }));
 			void dispatch(projectActions.loadAllContributorsByProjectId(projectId));
+
+			const todayDate = new Date();
+			const endDate = todayDate.toISOString();
+			const startDate = subtractDays(
+				todayDate,
+				ANALYTICS_DATE_MAX_RANGE,
+			).toISOString();
+			void dispatch(
+				projectActions.loadAllContributorsActivityByProjectId({
+					endDate,
+					projectId,
+					startDate,
+				}),
```

Comment:
Maybe I should dispatch this action inside `projectActions.loadAllContributorsByProjectId` action, or even rid of this action and place its code into mensioned action, cause in case when loading contributors we also want their activity being loaded? or it's better to keep them separate.

Reply:
I think we need to dispatch it inside loadAllContributorsByProjectId, it's ok to keep it as a separate action